desktop/libs/hadoop/src/hadoop/fs/upload.py

```diff
@@ -51,6 +56,86 @@
 UPLOAD_SUBDIR = 'hue-uploads'
 
 
+class LocalFineUploaderChunkedUpload(object):
+  def __init__(self, request, *args, **kwargs):
+    self._request = request
+    self.qquuid = kwargs.get('qquuid')
+    self.qqtotalparts = kwargs.get('qqtotalparts')
+    self.totalfilesize = kwargs.get('qqtotalfilesize')
+    self.file_name = kwargs.get('qqfilename')
+    local = "local:/"
+    if local in kwargs.get('dest', ""):
+      self.dest = kwargs.get('dest')[len(local):]
+    else:
+      self.dest = kwargs.get('dest')
+    self.file_name = kwargs.get('qqfilename')
+    self.filepath = request.fs.join(self.dest, self.file_name)
+    self._file = None
+    self.chunk_size = 0
+  def check_access(self):
+    pass
+  def upload_chunks(self):
+    pass
+  def upload(self):
+    self.check_access()
+    self.upload_chunks()
+
+
+class HDFSFineUploaderChunkedUpload(object):
```

Comment:
Same comments for Popupexception and Log messaged and better variable naming just like for new OFS handler above.